module/VuFind/src/VuFind/View/Helper/Root/Record.php

```diff
@@ -68,6 +72,13 @@ class Record extends AbstractClassBasedTemplateRenderer
      */
     protected $config;
 
+    /**
+     * Cover loader
+     *
+     * @var CoverLoader
+     */
+    protected $coverLoader;
```

Comment:
Since this is not being injected through the constructor, it should probably be initialized to null, and safety checks put in place to avoid trying to call methods on it if it is not populated correctly... or, if practical, maybe it should be refactored to a constructor parameter to guarantee its presence.